### PR TITLE
Create Radioreference.com.xml

### PR DESCRIPTION
**Pull request by phuzion**
Should solve #16733.

### CODE REVIEW

File: src/chrome/content/rules/Radioreference.com.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="Radioreference.com">
+	<target host="radioreference.com" />
+	<target host="www.radioreference.com" />
+	<target host="api.radioreference.com" />
+	<target host="forums.radioreference.com" />
+	<target host="m.radioreference.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Radioreference.com.xml]: `proxy.radioreference.com`

File: src/chrome/content/rules/Radioreference.com.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<!--
+	Nonfunctional hosts in *.radioreference.com:
+
+	h: http redirect
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
```
Comment (J0WI) [inline src/chrome/content/rules/Radioreference.com.xml]: You can remove this comments.

Comment (phuzion) [reply src/chrome/content/rules/Radioreference.com.xml]: I've removed these in the latest commit.

File: src/chrome/content/rules/Radioreference.com.xml (hunk 1 above)
Comment (phuzion) [reply src/chrome/content/rules/Radioreference.com.xml]: proxy.radioreference.com is NXDOMAIN according to what I've found. Would you still like me to add it to the ruleset?

Comment (J0WI) [reply src/chrome/content/rules/Radioreference.com.xml]: No, you can ignore this.

File: src/chrome/content/rules/Radioreference.com.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<ruleset name="Radioreference.com">
+	<target host="radioreference.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Radioreference.com.xml]: Connection times out

Comment (phuzion) [reply src/chrome/content/rules/Radioreference.com.xml]: I've removed the root domain, as it times out on HTTP and HTTPS.

Comment (J0WI) [reply src/chrome/content/rules/Radioreference.com.xml]: Thanks!

**Pull request merged**